### PR TITLE
Proper package versions

### PR DESCRIPTION
**Pull request by Adaickalavan**
Currently, doing
```
$ cd /path/to/SMARTS
$ git checkout develop   (commit: 555b1de)
$ python3.7 -m venv ./.venv
$ source ./.venv/bin/activate
$ pip install --upgrade pip
$ pip install -e .[train,test,dev,camera-obs]
$ scl scenario build-all ./scenarios
```
throws the following 
```
...
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/kyber/workspaces/smarts-second/cli/studio.py", line 77, in _build_single_scenario
    generate_glb_from_sumo_network(map_net, str(map_glb))
  File "/home/kyber/workspaces/smarts-second/smarts/sstudio/sumo2mesh.py", line 27, in generate_glb_from_sumo_network
    road_network.to_glb(out_glb_file)
  File "/home/kyber/workspaces/smarts-second/smarts/core/sumo_road_network.py", line 201, in to_glb
    glb = self._make_glb_from_polys(polys)
  File "/home/kyber/workspaces/smarts-second/smarts/core/sumo_road_network.py", line 966, in _make_glb_from_polys
    return _GLBData(gltf.export_glb(scene, extras=metadata, include_normals=True))
TypeError: export_glb() got an unexpected keyword argument 'extras'
...
```
due to a mismatch in `trimesh` package version.

This pull request 
1. updates packages in `setup.py` and `requirements.txt` to appropriate versions
2. fixes `test_pip_packages.sh` 
3. removes unnecessary pip install in ci tests

### CODE REVIEW

File: utils/setup/test_pip_packages.sh [hunk 1]
```diff
@@ -3,11 +3,10 @@
...
 python3.7 -m venv ${venv_dir}
 . ${venv_dir}/bin/activate
-
-# Install our requirements
 pip install --upgrade pip
-pip install -r requirements.txt
 
-# Install our package on top
+# Install minimum packages
 pip install .[test,train,camera-obs]
```
Comment (Gamenot) [inline utils/setup/test_pip_packages.sh:9]: I would not say these are minimum packages but they are definitely the test suite packages.

**Review by Gamenot — APPROVED** [review]

File: .github/workflows/ci-pull-request.yml [hunk 2]
```diff
@@ -22,7 +22,6 @@ jobs:
           . ${{env.venv_dir}}/bin/activate
           pip install --upgrade pip
           pip install wheel
```
Comment (saulfield) [inline .github/workflows/ci-pull-request.yml:24]: Should this have an `--upgrade` flag as well? Same with most of the other yml files

**Review by saulfield — APPROVED** [review]

File: utils/setup/test_pip_packages.sh (hunk 1 above)
Comment (Adaickalavan) [reply utils/setup/test_pip_packages.sh:9]: Rewrote the comment.

File: .github/workflows/ci-pull-request.yml (hunk 2 above)
Comment (Adaickalavan) [reply .github/workflows/ci-pull-request.yml]: Included `pip install --upgrade wheel` .

**Pull request merged**